invokeai/backend/model_manager/probe.py

```diff
@@ -747,8 +756,27 @@ def get_base_type(self) -> BaseModelType:
 
 
 class T5EncoderFolderProbe(FolderProbeBase):
+    def get_base_type(self) -> BaseModelType:
+        return BaseModelType.Flux
```

Comment:
Prior to this PR we've been installing these with a base type of Any. Do you have strong feelings that this should be set to flux here? I think it's possible they could be used for other pipelines in the future.